hail/src/test/scala/is/hail/nativecode/NativeDecoderSuite.scala

```diff
@@ -82,7 +82,7 @@ class NativeDecoderSuite extends HailSuite {
            |auto jis = h->at(0);
            |
            |auto is = std::make_shared<InputStream>(up, jis);
-           |${ spec.nativeInputBufferType } leb_buf {is};
+           |${ spec.nativeInputBufferType _ } leb_buf {is};
```

Comment:
I'm confused. Shouldn't this be `spec.nativeInputBufferType("InputStream")`?

Reply:
yep. bad rebase.